app/data/models/payment-request.js

```diff
@@ -20,6 +20,18 @@ module.exports = (sequelize, DataTypes) => {
     invoiceCorrectionReference: DataTypes.STRING,
     value: DataTypes.DECIMAL,
     received: DataTypes.DATE,
+    receivedFormatted: {
```

Comment:
Again, big fan of this and would encourage use of

E.g. could we do this for debt/values to map 7000 => 7000.00 and 342.2 => 342.20

Reply:
Cool